ts/components/conversation/media-gallery/EmptyState.md

```diff
@@ -0,0 +1,5 @@
+```js
+<div style={{position: 'relative', width: '100%', height: 300}}>
+  <EmptyState label="You have no attachments with media"/>
+</div>
+```
```

Comment:
For the potential permutations we might want for this component, what if we had multiple different parent window sizes?

Reply:
Good idea. Added another height for the ‘documents’ empty state.